asserts/ifacedecls.go

```diff
@@ -235,10 +235,17 @@ func (c *AttributeConstraints) Check(attrs map[string]interface{}) error {
 	return c.matcher.match("", attrs)
 }
 
+// OnClassicConstraint specifies a constraint based whether the system is classic and optional specific distros' sets.
+type OnClassicConstraint struct {
+	Classic bool
+	Distros []string
```

Comment:
What we have elsewhere is os.ID.. which unfortunately would be an awkward exported field name as "OSID". Considering this won't be seen anywhere outside of here, that sounds okay.